doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
 weight: 3
 ---
 
-To help you get started with Materialize, we'll:
+This guide walks you through getting started with Materialize, from installing it to creating your first materialized view on top of streaming data. We'll cover:
 
-- Install, run, and connect to Materialize
-- Explore its API
-- Set up a real-time stream to perform aggregations on Wikipedia user edits.
+* Installing, running and connecting to Materialize
+
+* Connecting to a streaming data source
+
+* Creating a materialized view
+
+* Exploring common patterns like joins and time-windowing
+
```

Comment:
Let's break this out into two bullets:

* Trying out joins
* Exploring time-windowed computations, such as "Updates from the last ten minutes" or "Orders from the last day, broken out by hour".